$OpenBSD: patch-gettext-tools_src_msgmerge_c,v 1.3 2019/05/20 23:45:48 naddy Exp $

Fix behaviour of --for-msgfmt on PO files with no translations.
https://git.savannah.gnu.org/gitweb/?p=gettext.git;a=commitdiff;h=2336451ed68d91ff4b5ae1acbc1eca30e47a86a9

Index: gettext-tools/src/msgmerge.c
--- gettext-tools/src/msgmerge.c.orig
+++ gettext-tools/src/msgmerge.c
@@ -520,8 +520,8 @@ There is NO WARRANTY, to the extent permitted by law.\
   else
     {
       /* Write the merged message list out.  */
-      msgdomain_list_print (result, output_file, output_syntax, force_po,
-                            false);
+      msgdomain_list_print (result, output_file, output_syntax,
+                            for_msgfmt || force_po, false);
     }
 
   exit (EXIT_SUCCESS);
